08.ls_object/ls_formatter.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class LsFormatter
+  COLUMNS = 3 # 3列表示
+
+  def initialize(contents, xattr_files_set = nil)
```

Comment:
```suggestion
  def initialize(contents, xattr_files_set)
```

LsFormatter.newを引数1個で呼ぶケースがないので必須にしてしまえばいいのでは？と思いました。